libs/barista-components/breadcrumbs/src/breadcrumbs-item.ts

```diff
@@ -27,10 +31,37 @@ import { Directive, ElementRef } from '@angular/core';
   exportAs: 'dtBreadcrumbsItem',
   host: {
     class: 'dt-breadcrumbs-item',
+    '[class.dt-breadcrumbs-item-non-interactive]': '!_isFocusable',
+    '(keydown)': '_onKeyDown($event)',
   },
 })
-export class DtBreadcrumbsItem2 {
-  constructor(private readonly _elementRef: ElementRef<HTMLAnchorElement>) {}
+export class DtBreadcrumbsItem2 implements Highlightable {
+  constructor(
+    readonly _elementRef: ElementRef<HTMLAnchorElement>,
+    private _interactivityChecker: InteractivityChecker,
```

Comment:
These changes in the constructor are also considered a breaking change. Please annotate them accordingly